packages/components/cascader-panel/src/node.ts

```diff
@@ -190,7 +190,7 @@ class Node {
 
   onChildCheck() {
     const { children } = this
-    const validChildren = children.filter((child) => !child.isDisabled)
+    const validChildren = children.filter((child) => !child.isDisabled || child.checked)
```

Comment:
This seems to solve the problem, but there seems to be room for optimization in terms of code logic and reading.
The `checked` status verification of the parent node should only depend on whether all the child nodes are selected, and the `checkStrictly` scenario must be considered.